modules/ddl-and-loading/pages/add-token-function.adoc

```diff
@@ -47,50 +33,52 @@ below depending on the attribute type.
 |`void`. The value of `oToken` will be returned in GSQL
 
 |`BOOL`
-|`extern "C" bool funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum) `
+|`extern "C" bool funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum)`
 |`bool`
 
 |`UINT`
-|`extern "C" uint64_t funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum) `
+|`extern "C" uint64_t funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum)`
 |`uint64_t`
 
 |`INT`
-|`extern "C" int64_t funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum) `
+|`extern "C" int64_t funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum)`
 |`int64_t`
 
 |`FLOAT`
-|`extern "C" float funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum) `
+|`extern "C" float funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum)`
 |`float`
 
 |`DOUBLE`
-|`extern "C" double funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum) `
+|`extern "C" double funcName (const char* const iToken[], uint32_t iTokenLen[], uint32_t iTokenNum)`
 |`double`
 |===
 
-The parameters `+iToken+`, `+iTokenLen+`, and `+iTokenNum+` must be
+The parameters `iToken`, `iTokenLen`, and `iTokenNum` must be
 named exactly as such, and are used to describe the input tokens:
 
-* `+iToken+` is an array of the string tokens,
-* `+iTokenLen+` is an array of the lengths of the string tokens
-* `+iTokenNum+` is the number of tokens.
+* `iToken` is an array of the string tokens
+* `iTokenLen` is an array of the lengths of the string tokens
+* `iTokenNum` is the number of tokens
 
-For token functions for attribute types `+STRING+` or
-`+STRING COMPRESS+`, the return type for the C++ function is `+void+`.
-Use the parameter `+oToken+` to store the string you want returned, and
-in GSQL the token function will return the string stored in `+oToken+`:
+Token functions for attribute types `STRING` or
+`STRING COMPRESS` have a C{pp} function return type of `void`.
+Use the parameter `oToken` to store the string you want returned, and
+in GSQL the token function will return the string stored in `oToken`:
 
-* `+oToken+` is the returned string value
-* `+oTokenLen+` is the length of the return string
+* `oToken` is the returned string value
+* `oTokenLen` is the length of the return string
 
-Note that the input tokens are always in string (`+char*+`) format. If
+Note that the input tokens are always in string (`char*`) format. If
 necessary, convert them to other types inside the function.
 
```

Comment:
Did we add a note to say that users must make sure that the tokenbank functions compile, otherwise it impacts all loading jobs? I think that was the original suggestion from support.